demo/KevinDemo/Stories/AccountLinking/AccountLinkingView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AccountLinkingView.swift
+//  KevinDemo
+//
+//  Created by Daniel Klinge on 10/06/2022.
+//
+
+import SwiftUI
+
+public struct AccountLinkingView: View {
+    
+    @StateObject var viewModel = AccountLinkingViewModel()
+    
+    public var body: some View {
+        NavigationView {
+            ZStack {
+                Color("PrimaryBackgroundColor").ignoresSafeArea()
+
+                if viewModel.viewState.isLoading {
+                    ProgressView()
+                        .zIndex(2)
+                } else if viewModel.viewState.linkedBanks?.isEmpty == false {
+                    AccountsView(
+                        linkedBanks: viewModel.viewState.linkedBanks!.toArray(),
```

Comment:
If you apply the change on line 22, then you can do this here `linkedBanks: linkedBanks.toArray(),`